src/core/hle/service/acc/acc.cpp

```diff
@@ -504,16 +489,42 @@ class IGuestLoginRequest final : public ServiceFramework<IGuestLoginRequest> {
     }
 };
 
+class EnsureTokenIdCacheAsyncInterface final : public IAsyncContext {
+public:
+    explicit EnsureTokenIdCacheAsyncInterface(Core::System& system_) : IAsyncContext(system_) {
+        MarkComplete();
+    }
+    ~EnsureTokenIdCacheAsyncInterface() = default;
+
+    void LoadIdTokenCache(Kernel::HLERequestContext& ctx) {
+        LOG_WARNING(Service_ACC, "(STUBBED) called");
+
+        IPC::ResponseBuilder rb{ctx, 2};
+        rb.Push(ResultSuccess);
+    }
+
+protected:
+    bool IsComplete() override {
+        return true;
+    }
+
+    void Cancel() override {}
+
+    ResultCode GetResult() override {
+        return ResultSuccess;
+    }
+};
+
 class IManagerForApplication final : public ServiceFramework<IManagerForApplication> {
 public:
     explicit IManagerForApplication(Core::System& system_, Common::UUID user_id_)
-        : ServiceFramework{system_, "IManagerForApplication"}, user_id{user_id_} {
+        : ServiceFramework{system_, "IManagerForApplication"}, user_id{user_id_}, system(system_) {
```

Comment:
```suggestion
        : ServiceFramework{system_, "IManagerForApplication"}, user_id{user_id_} {
```
system is already a protected member of ServiceFramework